ff-asn-no-detailedLinks/index.html

```diff
@@ -0,0 +1,348 @@
+                               ssstyle="position: absolute; right: 15px;top:12px;">
+                    </div>
+
+                    <ff-slider-one-touch></ff-slider-one-touch>
+                </ff-slider-control>
+            </ff-asn-group-slider>
+
+            <ff-asn-group-slider>
+                <div slot="groupCaption" class="expand-arrow"><div>{{group.name}}</div></div>
+
+                <div data-container="removeFilter">
+                    <svg viewBox="3 3 10 10"><use xlink:href="#icon-cross"></use></svg>
+                    <span>Reset Filter</span>
+                </div>
+
+                <ff-slider-control submit-on-input="true">
+                    <input data-control='1'
+                           style="position: absolute; left: 15px;top:12px;">
+                    <input data-control='2'
+                           style="position: absolute; right: 15px;top:12px;">
+
+                    <ff-slider current-min-value="0">
+                        <div slot="slider1"></div>
+                        <div slot="slider2"></div>
+                    </ff-slider>
+                </ff-slider-control>
+            </ff-asn-group-slider>
+
+        </ff-asn>
+    </div>
+    <div class="products-listing">
+        <ff-record-list unresolved>
+            <ff-record>
+                <a data-anchor="https://www.alpinetrek.co.uk{{record.Deeplink}}"
+                   data-redirect="https://www.alpinetrek.co.uk{{record.Deeplink}}" data-redirect-target="_blank">
+                    <div class="record-img">
+                        <img data-image="https://showcase.ff-labs.de/demoshop-images/{{record.ImageName}}">
+                    </div>
+                    <div class="record-product-info">
+                        <h3 class="record-product-name">
+                            {{record.Title}}
+                        </h3>
+                        <div class="record-product-brand">{{record.Manufacturer}}</div>
+                        <div class="record-product-price">{{record.Price}}</div>
+                    </div>
+                </a>
+                <div class="record-product-add-to-cart">
+                    <button>Add to Cart</button>
+                </div>
+            </ff-record>
+        </ff-record-list>
+    </div>
+</main>
+<script>
+    const ffAsn = document.querySelector('ff-asn');
+    ffAsn.addEventListener('dom-updated', () => {
+        ffAsn.querySelectorAll('ff-asn-group').forEach(ffAsnGroup => {
+            // TMP - simulates "Max. number of displayed filter elements" = 0 filter setting - remove after a proper FF setup is ready
+            if (ffAsnGroup.group.name === "Color") ffAsnGroup.group.detailedLinks = 0;
+
+            if (ffAsnGroup.group.detailedLinks === 0 && ffAsnGroup.group.selectedElements.length === 0) {
+                ffAsnGroup.group = {...ffAsnGroup.group, detailedLinks: 9999};
+                setTimeout(() => ffAsnGroup.hide(false));
```

Comment:
Do you think there is a feasible way to hide the group conditionally? If you expand the group, select a filter, unselect it again, the group will be collapsed again.
I think it would be more intuitive if manual expansion overrides the autocollapse.